firebase_admin/db.py

```diff
@@ -125,40 +125,91 @@ def child(self, path):
         full_path = self._pathurl + '/' + path
         return Reference(client=self._client, path=full_path)
 
-    def get(self):
-        """Returns the value at the current location of the database.
+    def _get_with_etag(self):
```

Comment:
How about we just move this logic straight into the `get()` method, and get rid of this helper? `transaction()` can call `self.get(etag=True)`.